build_tools/travis/before_install.sh

```diff
@@ -28,6 +28,10 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
         brew install vtk
     fi
 
+    if [[ "$USE_OMP" == 1 ]]; then
+        brew install llvm
```

Comment:
ok I tested. It does the job but you need to change the compiler that cmake is using.

for me I need to set:

CPP = /usr/local/opt/llvm/bin/clang
CPPFLAGS = -I/usr/local/opt/llvm/include -fopenmp
LDFLAGS = -L/usr/local/opt/llvm/lib

this being said we should not ship binaries with clang-omp / llvm on mac as it will break on most systems that don't brew setup properly.

We should ship non-openmp binaries on mac and update the brew formula to add an openmp option.

So to sum up:
- setup/update only one travis build so it uses openmp using brew setting up properly all env vars.
- upload to gforge non-openmp binaries for our users